src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -794,20 +797,18 @@ public enum LegendsAwakened implements LogicCardInfo {
           weakness P, '+20'
           pokePower "Time Walk", {
             text "Once during your turn, when you put Azelf from your hand onto your Bench, you may look at all of your face-down Prize cards. If you do, you may choose 1 Pokémon you find there, show it to your opponent, and put it into your hand. Then, choose 1 card in your hand and put it as a Prize card face down."
-            actionA {
-              assert my.hand : "No cards in hand"
-              checkLastTurn()
-              powerUsed()
-
-              def newPrize = my.hand.select(hidden: true, "Card to put into Prizes").first()
-
-              def tar = my.prizeCardSet.faceDownCards.select(hidden: false, "Choose a Prize card to replace with one in your hand.").first()
-              my.hand.add(tar)
-
-              def indexOfOldPrize = my.prizeCardSet.indexOf(tar)
-              my.prizeCardSet.set(indexOfOldPrize, newPrize)
-              my.prizeCardSet.setVisible(newPrize, true)
-              my.hand.remove(newPrize)
+            onActivate {r->
+              if (r==PLAY_FROM_HAND && confirm("Use Time Walk?")){
```

Comment:
Not sure if this one matters in the actual Play server, but on the Dev server at least it'd be kind of annoying to place this Pokémon as the "active" as the first Pokémon and be asked this.
```suggestion
              if (r==PLAY_FROM_HAND && self.benched && confirm("Use Time Walk?")){
```